tests/geometry/point.cpp

```diff
@@ -1,6 +1,18 @@
 #include <geometry/point.h>
+#include <geometry/constants.h>
+#include <assert.h>
+#include <cmath>
 using namespace std;
 
 int main(){
-    Point p(-2, -1, 0);
+    Point p1(1, 2, 3), p2(4, 5, 6);
+
+    assert(abs(p1.distance(p2) - 5.196152) < EPSILON);
+    assert(abs(p2.distance(Point(0, 0, 0)) - 8.774964) < EPSILON);
```

Comment:
aqui tu já não precisa usar o EPSILON do que tá definido no `geometry/utils.h`. Aqui como são valores que tu botou na mão, pode botar teu próprio epsilon.

Reply:
Nos testes em que eu uso um **epsilon**, eu removi o import do antigo `geometry/constants.h` e declarei um `const double epsilon = ...;` logo no início do código, para facilitar a visualização